app/models/dialog_field_radio_button.rb

```diff
@@ -4,6 +4,20 @@ def show_refresh_button?
   end
 
   def initial_values
-    [["", "<None>"]]
+    [[nil, "<None>"]]
+  end
+
+  private
+
+  def raw_values
+    @raw_values ||= dynamic ? values_from_automate : static_raw_values
+
+    self.value ||= default_value if @raw_values.collect { |value_pair| value_pair[0] }.include?(default_value)
```

Comment:
@eclarizio Would suggest pulling this logic out into a separate private method with a descriptive name.  Would make the `raw_values` method easier to read and make the logic of this line clearer as well.